circle.yml

```diff
@@ -41,17 +41,6 @@ test:
     # behind a user namespace which Circle CI tests are run under
     - sudo env "PATH=$PATH" bin/yarn install --force
 
-deployment:
-  release:
-    tag: /v[0-9]+(\.[0-9]+)*/
-    owner: yarnpkg
-    commands:
-      # Only NPM is handled here - All other release files are handled in a webhook.
-      - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-      - ./scripts/set-installation-method.js "`pwd`/package.json" npm
-      # https://github.com/yarnpkg/yarn/issues/610, need to make yarn publish noninteractive
-      - npm publish --tag rc
```

Comment:
I agree with @BYK.
The advantage of Circle CI builds is that they are simple, easy to debug and maintain.
Moving everything to custom jenkins bot may be more cohesive and safe but the support becomes depending 99% on you, @Daniel15